package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@editorjs/nested-list",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Nested list Tool for EditorJS",
   "main": "dist/nested-list.js",
   "repository": "https://github.com/editor-js/nested-list.git",
   "author": "CodeX <team@codex.so>",
   "license": "MIT",
   "scripts": {
+    "prepare": "npm run build",
```

Comment:
This includes a previous change to add "prepare" for installing from github. Not sure how to exclude it from this PR.

Reply:
you can simply remove this line

Reply:
I don't understand how.

I have my own repo, with the changes I need for things to work for me. It has two changes: 1) prepare, 2) nested list onPaste. I don't want to remove 1 from my repo. Can I just remove it from this PR without removing it from my repo?

Reply:
Ideally you can accept #23 and then I can rebase.

Reply:
Ah, I could make a new branch, cherry pick the changes onto that branch, and make a PR of it. http://blog.asquareb.com/blog/2014/06/19/making-a-git-pull-request-for-specific-commits/

However, it would be easier if you are willing to accept #23.